src/test/java/com/team4/testingsystem/controllers/FileAnswerControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.entities.FileAnswer;
+import com.team4.testingsystem.entities.Question;
+import com.team4.testingsystem.exceptions.NotFoundException;
+import com.team4.testingsystem.repositories.FileAnswerRepository;
+import com.team4.testingsystem.repositories.QuestionRepository;
+import com.team4.testingsystem.services.FileAnswerService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class FileAnswerControllerTest {
+
+    @Mock
+    private QuestionRepository questionRepository;
+
+    @Mock
+    private FileAnswerRepository fileAnswerRepository;
+
+    @Mock
+    private FileAnswerService fileAnswerService;
+
+    @Spy
+    private FileAnswer fileAnswer;
+
+    @InjectMocks
+    private FileAnswerController fileAnswerController;
+
+    @Test
+    void getSuccess() {
+        Mockito.when(fileAnswerController.get(Mockito.anyLong())).thenReturn(fileAnswer);
+        Assertions.assertEquals(fileAnswer, fileAnswerController.get(Mockito.anyLong()));
```

Comment:
can you describe, what are you trying to test here? If I got it correct:
1) you force fileAnswerController return fileAnswer on get request with any long
2) you are checking that p.1 works.

Are you trying to test Mockito?

Reply:
yes, I tested it. There should not be any long, but only one value.